### ISSUE TITLE
How to Update Language in Intro Paragraph?

### ISSUE THREAD

**Issue by bfreeds** [cascobay/cascobay#22]
From @sarahllyman:

> The years the data was collected needs to be updated and we'd like to move the "Click on any dot in the map to learn about that site and see its score." language to the top.

**Comment by bfreeds** [comment]
@sarahllyman - I see in commit https://github.com/cascobay/cascobay/commit/64c4e2cb6c1abefc4c6487b73557a29ef916fe0d how you tried to change the language for the intro paragraph.

The reason you are not seeing the language updated on the webpage is that there are two branches of code, `master` and `gh-pages`.  The change you made is on the `master` branch.  The website is being hosted by Github through the `gh-pages` branch.  `Master` is the branch in which all the code is edited and developed, but there is a build step that translates what is in `master` into the live `gh-pages` branch.  I apologize for the confusion - being the first web project I've worked on, I underestimated the difficulty in managing changes when passing it off to you.  Happy to help make changes before the launch.  I'll also document how I build from the `master` to `gh-pages` branch in this task to show how it is done.
